ArjunaJTS/jtax/classes/com/arjuna/ats/internal/jta/transaction/jts/TransactionImple.java

```diff
@@ -1756,13 +1758,20 @@ private final Xid createXid (boolean branch, XAModifier theModifier)
 	{
 		Xid jtaXid = baseXid();
 
-		// if Xid is from Narayana (format corresponds with Narayana JTS xid we allow Xid to be edited)
+		// if Xid does not belong to Narayana not allowing to be edited
```

Comment:
that's a good point the word JTS should be left in the comment. The comment was rephrased because I moved the if curly braces down to the try. The text now correspond with the functionality covered by the if.